### ISSUE TITLE
Improve accessibility using automation

### ISSUE THREAD

**Issue by lukebrody** [kavigupta/urbanstats#817]
e.g. it should be relatively easy to have some sort of automation to ensure alt tags, etc

Research what's important for accessibility, maybe get feedback from users

**Comment by lukebrody** [comment]
Design:

- Use https://github.com/dequelabs/axe-core-npm
  - Include the axe.min.js when in dev or CI mode, conditionally in build.py
  - Serve axe.min.js directly from cdnjs (using hash to verify integrity)
    - Downside: a11y testing doesn't work offline
    - Upside: Reduces complexity around bundling/serving the axe.min.js
- When running e2e tests, when a screenshot is taken, use `axe` to analyze using `ClientFunction` and return a report.
  - Output the report to a text file artifact named similarly to screenshots, formatted using code similar to [this](https://github.com/dequelabs/axe-core-npm/blob/5ef380969997863a40da9fb3205e8ec4ea4f1332/packages/cli/src/lib/events.ts#L42)
  - For now, show a warning when there are a11y problems. Slowly backfill fixes, and then turn on enforcement, failing the test. (Consider only failing at the end of the suite, so that tests can still run)
